app/scripts/services/tableinfo.js

```diff
@@ -7,9 +7,16 @@ const tableinfo = angular.module('tableinfo', ['sql'])
     var isActiveShard = function(shard) {
       return ACTIVE_SHARDS.indexOf(shard.routing_state) > -1;
     };
+
+    var dotNotationToSubscript = function (column) {
+      return column.replace(/\.([a-zA-Z]+)/, function(match, p1, offset, str) {
+        return `['` + p1 + `']`; 
+      });
```

Comment:
what about nested subscript expression like `pk.a.b`? looks to as this still wouldn't work correctly

Reply:
Ah right, pushed a fixup:

```
"obj.a.b.c".replace(/\.([a-zA-Z]+)/g, function(match, p1, offset, str) {
        return `['` + p1 + `']`; 
      })

→

"obj['a']['b']['c']"
```